src/App.js

```diff
@@ -1,13 +1,16 @@
 import "./App.css";
 import logo from "./logo.png";
 import LandingPage  from "./pages/landing-page/landingPage.jsx";
-import  { Nav } from "./comps/navigation/navigation.jsx"
+import  { Nav } from "./comps/navigation/navigation.jsx";
+import { ProductPage} from "./pages/product-page/productPage.jsx"
```

Comment:
you can import landing page and product page from a single file, 
[check here](https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno)